### DISCUSSION TITLE
Any way to disable the document title getting updated by default?

### DISCUSSION THREAD

**Discussion by rungta** [bigskysoftware/htmx#1756] [Q&A] [answered]
Hi. We’re using htmx to lazyload content from other pages on some modals. We are not modifying the HTML response in any way, but instead using `hx-select` to cherry-pick the relevant modal content from the full page response. However, an unintended side-effect of this is that the original page’s `<title>` gets updated by htmx. Is there any way to disable this behaviour? I could not spot a config setting.

**Comment by Telroshan** [comment, marked as answer]
Hey, not currently, but this is actually being worked on by @1cg on [the dev branch](https://github.com/bigskysoftware/htmx/commit/c230931d42489a629d685bedb440fadd7eaf1416), so I think you may expect that `ignoreTitle` feature in the next release, it'll look like this:
```html
<button hx-get="/test" hx-swap="innerHTML ignoreTitle:true">Click Me!</button>
```
